detox/ios/Detox/GREYActions+Detox.h

```diff
@@ -0,0 +1,16 @@
+//
+//  GREYActions+Detox.h
+//  Detox
+//
+//  Created by Matt Findley on 2/7/19.
+//  Copyright © 2019 Wix. All rights reserved.
+//
+
+@import Foundation;
+#import <EarlGrey/EarlGrey.h>
+
+@interface GREYActions (Detox)
+
++ (id<GREYAction>)detoxSetDatePickerDateIOSOnly:(NSString *)dateString withFormat:(NSString *)dateFormat;
```

Comment:
Let's not call this "IOSOnly". Should be `detoxSetDatePickerDate:withFormat:`.

Reply:
📗 1d6913c